apps/autoscheduler/FunctionDAG.h

```diff
@@ -185,24 +185,24 @@ class LoadJacobian {
 
     void dump(const char *prefix) const {
```

Comment:
I don't think the bodies of any of the dump calls should be changed in this PR, but rather the call to dump should be conditional on the debug level

Reply:
(I'm imagining adding new dump calls while debugging and getting confused regarding why things aren't being dumped)

Reply:
done, I think